code/WorkInProgress/laundry.dm

```diff
@@ -2,6 +2,8 @@
 #define WASH "w"
 #define DRY "d"
 #define POST 1
+#define cycle_time_mob_inside 5
+#define cycle_time 10
```

Comment:
```suggestion
#define cycle_time_mob_inside 5
#define cycle_time 10
```

These should be capitalized (standard for defines). Otherwise seemsgood

Reply:
Capitalized them.